js-intro.js

```diff
@@ -12,32 +12,61 @@ var temp2 = 350
 var temp3 = 212
 
 
+// create a function that takes in one argument
+// tells if the argument is above, below, or at 212
+const tempTest = (temp) => {
+    if (temp > 212) {
+        return `${temp} is above the boiling point.`
+    } else if (temp < 212) {
+        return `${temp} is below the boiling point.`
+    } else if (temp === 212) {
+        return "212 is at the boiling point."
+    }
+}
+console.log(tempTest(35));
+console.log(tempTest(350));
+console.log(tempTest(212));
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
-
-
+// create a function that takes in an arrray and multipies it by 5
+const multipier = (array) => {
+    let newArray = []
+    for (let i = 0; i < array.length; i++) {
+    newArray.push(array[i] * 5)
+    }
+    return newArray
+}
+console.log (multipier(myNumbers1))
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+const mapMultipier = (array) => {
+    let newArray = array.map (value => value * 5)
```

Comment:
You can save yourself a line of code by returning your mapped array outright, rather than creating a new variable in order to turn around and immediately return it. Map is an accessor which doesn't permanently change to original array, so if you wanted to use those values later in your code you would need to save it to a new variable like you did. But for this purpose, you can save yourself some key strokes :)